rs/ethereum/cketh/minter/src/deposit.rs

```diff
@@ -238,11 +239,15 @@ where
         let range = subranges.pop_front().unwrap();
         let (from_block, to_block) = range.clone().into_inner();
 
-        let request = GetLogsParam {
-            from_block: BlockSpec::from(from_block),
-            to_block: BlockSpec::from(to_block),
-            address: vec![contract_address],
-            topics: topics.clone(),
+        let request = GetLogsArgs {
+            from_block: Some(BlockTag::Number(Nat256::from_be_bytes(
+                from_block.to_be_bytes(),
+            ))),
```

Comment:
nit: There is already a conversion between `Nat256` and `CheckedAmountOf<Unit>`
```suggestion
            from_block: Some(BlockTag::Number(Nat256::from(from_block))),
```

Reply:
Thanks for catching that!